.travis.yml

```diff
@@ -30,7 +32,9 @@ branches:
 # Clones WordPress and configures our testing environment.
 before_script:
     - export SLUG=$(basename $(pwd))
-    - svn co --quiet http://develop.svn.wordpress.org/tags/$WP_VERSION $WP_CORE_DIR
+    - export WP_TESTS_DIR=/tmp/wordpress/tests/phpunit
+    - export WP_CORE_DIR=/tmp/wordpress/
+    - git clone --depth=1 --branch="$WP_VERSION" git://develop.git.wordpress.org/ $WP_CORE_DIR
```

Comment:
This is cool. The `#.#` branches point to the latest commit for their branch, not to a tagged release, so we don't have to worry about updating point releases.

I'm currently opposed to using `master`, though, because it looks like it's the WIP branch for the next release, and under active development. Is that the case?